core/services/mongodb-remote/src/main/java/com/mongodb/stitch/core/services/mongodb/remote/sync/internal/DataSynchronizer.java

```diff
@@ -1006,7 +1043,7 @@ private void resolveConflict(
     final boolean acceptRemote =
         (remoteEvent.getFullDocument() == null && resolvedDocument == null)
             || (remoteEvent.getFullDocument() != null
-            && transformedRemoteEvent.getFullDocument().equals(resolvedDocument));
+            && remoteEvent.getFullDocument().equals(resolvedDocument));
```

Comment:
[major] This is reintroducing a bug. Please undo any reformats or changes unrelated to this ticket in this PR.